docfx/doxygen2markdown.cc

```diff
@@ -414,7 +423,7 @@ bool AppendIfDocTitleCmdGroup(std::ostream& os, MarkdownContext const& ctx,
   // Unexpected: subscript, superscript, center, small, del, ins
   // Unexpected: htmlonly, manonly, rtfonly, latexonly, docbookonly
   // Unexpected: image, dot, msc, plantuml
-  // Unexpected: anchor
+  if (name == "anchor") return true;  // Generate nothing and succeed
```

Comment:
Opt: Treat this just like the others (e.g. `AppendIfAnchor()`), even if it appends nothing?

Reply:
Done.